apps/filebrowser/src/filebrowser/templates/listdir_components.mako

```diff
@@ -1854,6 +1860,18 @@ else:
           resetPrimaryButtonsStatus(); //globally available
           return false;
         }
+        if ($("#newDirectoryNameInput").val().length < 3 && (self.isOFSRoot() || self.isOFSVol())) {
+          $("#volumeBucketNameAlert").show();
+          $("#newDirectoryNameInput").addClass("fieldError");
+          resetPrimaryButtonsStatus(); //globally available
+          return false;
+        }
+        if (($("#newDirectoryNameInput").val() !== $("#newDirectoryNameInput").val().toLowerCase()) && (self.isOFSRoot() || self.isOFSVol())) {
```

Comment:
nit: You could consider surrounding both checks with one if for `self.isOFSRoot() || self.isOFSVol()`